aesara/tensor/random/basic.py

```diff
@@ -492,8 +492,8 @@ def rng_fn(cls, rng, n, a, b, size=None):
 class MultinomialRV(RandomVariable):
     """A Multinomial random variable type.
 
-    FYI: Support shape is determined by the first dimension in the *second*
-    parameter (i.e.  the probabilities vector).
+    FYI: Length of the support dimension is determined by the last
```

Comment:
Whats FYI mean?

Reply:
For your information?

Reply:
Got it. I suggest removing it for a couple of reasons then. A docstring is implicitly for your information so its not needed, but more importantly even as an english speaker this acronym gave me pause so it could be especially confusing for non english people.

Perhaps change it to a Notes section?
https://numpydoc.readthedocs.io/en/latest/format.html#notes

Reply:
I bet you by a few seconds there, sorry :p

Reply:
Haha no problem. Great minds think alike ;) 

Thanks for doing the real work of actually identifying and making the PR